### DISCUSSION TITLE
Issue with gui and id

### DISCUSSION THREAD

**Discussion by atschoolrn** [3kh0/ext-remover#635] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So, whenever i use either the GUI method or the =id? method, the extensions still work and show up as enabled. Is there any fix? And yes, i know, im on v106+, but im still curious if there is any method to exploit the new patch.

**Comment by piplup7575** [comment, marked as answer]
just use ltmeat
there is no way to use ltbeef past 106
